plugins/game/lib/api/filters/KalturaUserScorePropertiesFilter.php

```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+ * @package plugins.game
+ * @subpackage api.filters
+ * @abstract
+ * @relatedService UserScoreService
+ */
+class KalturaUserScorePropertiesFilter extends KalturaUserScorePropertiesBaseFilter
+{
+	/**
+	 * @var int
+	 */
+	public $gameObjectId;
+	
+	/**
+	 * @var KalturaGameObjectType
+	 */
+	public $gameObjectType;
+	
+	/**
+	 * @var string
+	 */
+	public $userIdEqual;
+	
+	/**
+	 * @var string
+	 */
+	public $userIdIn;
+	
+	/**
+	 * @var int
+	 */
+	public $placesAboveUser;
+	
+	/**
+	 * @var int
+	 */
+	public $placesBelowUser;
+	
+	public function getMapBetweenObjects()
+	{
+		return array_merge(parent::getMapBetweenObjects(), self::$map_between_objects);
+	}
+	
+	protected function getCoreFilter()
+	{
+		return null;
+	}
+	
+	protected function initGameServicesRedisInstance()
+	{
+		$redisWrapper = new kInfraRedisCacheWrapper();
+		$redisConfig = kConf::get('game', kConfMapNames::REDIS);
```

Comment:
what if there is no such section? or specific configuration option
need to handle errors